src/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/CSharpIdentifier.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Globalization;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X
+{
+    internal static class CSharpIdentifier
```

Comment:
Do we have a bug tracking moving this back into Razor.Langauge? We should do that.